xla/service/gpu/gpu_windowed_einsum_handler.cc

```diff
@@ -68,6 +68,16 @@ absl::Status UpdateDotAndConsumerConfig(HloInstruction* dot,
   return absl::OkStatus();
 }
 
+absl::Status SetForceDelayForInstruction(HloInstruction* instr,
+                                         bool force_delay) {
+  auto gpu_config = instr->backend_config<gpu::GpuBackendConfig>();
```

Comment:
This could be done in a separate PR?

Reply:
Sure, let me take the changes in this pass to a separate pr.